allennlp/data/vocabulary.py

```diff
@@ -9,26 +9,37 @@
 
 class _NamespaceDependentDefaultDict(defaultdict):
     """
-    Using ``defaultdicts`` to handle namespaces is really convenient - you don't need to worry
-    about whether you've seen a particular namespace before in any of the methods in
-    ``Vocabulary``.  But because some namespaces need padding (like "tokens") and some don't (like
-    "labels"), we want different defaults depending on the namespace.  This lets us still use a
-    ``defaultdict``, but have different default values depending on the key that we're given.
+    Sometimes certain namespaces need padding (like "tokens") and some don't (like
+    "labels"), and we want different defaults depending on the namespace.  This class lets us use a
+    ``defaultdict`` (https://docs.python.org/2/library/collections.html#collections.defaultdict),
```

Comment:
The URL should be inlined here: http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets.  Though I'm not sure we need the link.